contracts/LosslessReporting.sol

```diff
@@ -298,4 +298,19 @@ contract LosslessReporting is ILssReporting, Initializable, ContextUpgradeable,
         emit CompensationRetrieve(_adr, _amount);
     }
 
-}
+    /// @notice This function allows the admin to retrieve tokens that are stuck in the contract
+    /// @param _token Address of the token to be retrieved
+    /// @param _to Address to which the tokens will be sent
+    /// @param _amount Amount of tokens to be retrieved
+    function retrieveRejectedReportTokens(ILERC20 _token, address _to, uint256 _amount) public onlyLosslessAdmin {
```

Comment:
Function doesn't really check if the report is rejected. We should have some logic that would require that would check that tokens can be retrieved only when latest report is expired.

Reply:
Added check [0a874ca](https://github.com/Lossless-Cash/lossless-v3/pull/116/commits/0a874ca7d46994541403cffad25de32a06039f94)